### PR TITLE
BUG: Try to fix sdist command (fixes #177)

### LINKED ISSUE #177: Fix sdist
Currently sdist does not generate the correct archive for distribution. The important missing parts are:
1. `examples`
2. `integrationtests`
3. `docs`

Also need to ensure PKG-INFO is correct for upload to PyPi.

### PR DESCRIPTION
**Pull request by prabhuramachandran**
Have checked this with the `check-manifest` tool to make sure we are not
missing anything.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,23 @@
-prune artwork
+include .coveragerc .travis_before_install *.rst *.txt *.py
```
Comment (itziakos) [inline MANIFEST.in]: I do not think that one needs `.coveragerc` and especially  `.travis_before_install`

**Comment by itziakos** [comment]
just one comment, apart from that it looks ok

**Comment by dmsurti** [comment]
@prabhuramachandran I tried the sdist with upload to testpypi and the upload went fine.

https://testpypi.python.org/pypi/mayavi-dmsurti/4.4.3.dev0.

-
* PS: For test purpose, I changed the package name as testpyi already has a mayavi package under your ownership.

**Comment by prabhuramachandran** [comment]
I have updated the MANIFEST to not include the .coveragerc and the .travis... file.  @dmsurti please merge when the tests pass.  thanks.

**Pull request merged**